themes-default/slim/views/partials/showheader.mako

```diff
@@ -10,64 +10,47 @@
 <%namespace file="/inc_defs.mako" import="renderQualityPill"/>
 
 <div class="row">
-    <div id="showtitle" class="col-lg-12" data-showname="${show.name | h}">
+    <div id="showtitle" class="col-lg-12" :data-showname="show.title">
         <div>
-            <h1 class="title" data-indexer-name="${show.indexer_name}" data-series-id="${show.indexerid}" id="scene_exception_${show.indexerid}">
-                <app-link href="home/displayShow?indexername=${show.indexer_name}&seriesid=${show.indexerid}" class="snatchTitle">${show.name | h}</app-link>
+            <h1 class="title" :data-indexer-name="show.indexer" :data-series-id="show.id[show.indexer]" :id="'scene_exception_' + show.id[show.indexer]">
```

Comment:
```
:data-indexer-name="show.indexer"
:data-series-id="show.id[show.indexer]"
:id="'scene_exception_' + show.id[show.indexer]"
```

As far as I can tell, you can remove these three props.
The first two **are** used in [scene-exceptions-tooltip.js](https://github.com/pymedusa/Medusa/blob/feature/vueify-showheader/themes-default/slim/static/js/scene-exceptions-tooltip.js#L2-L4), however the file isn't imported anywhere ATM.
That file is used to put a 'qtip' tooltip on the show title, that lists the show's scene names.
If we want that, we should convert that to Vue **in a separate PR**.

Reply:
> If we want that, we should convert that to Vue in a separate PR.
Sounds good.